[2024-12-14T09:01:53.450+0530] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-14T09:01:53.468+0530] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: movie_pipeline.scrape backfill__2024-12-10T00:00:00+00:00 [queued]>
[2024-12-14T09:01:53.478+0530] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: movie_pipeline.scrape backfill__2024-12-10T00:00:00+00:00 [queued]>
[2024-12-14T09:01:53.479+0530] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-14T09:01:53.500+0530] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): scrape> on 2024-12-10 00:00:00+00:00
[2024-12-14T09:01:53.513+0530] {logging_mixin.py:190} WARNING - /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=30136) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-14T09:01:53.515+0530] {standard_task_runner.py:72} INFO - Started process 30163 to run task
[2024-12-14T09:01:53.515+0530] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'movie_pipeline', 'scrape', 'backfill__2024-12-10T00:00:00+00:00', '--job-id', '52', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/var/folders/h6/l2cjrfls77b6_h6zs5w5vn4r0000gn/T/tmpgyuwjeea']
[2024-12-14T09:01:53.521+0530] {standard_task_runner.py:105} INFO - Job 52: Subtask scrape
[2024-12-14T09:01:53.622+0530] {task_command.py:467} INFO - Running <TaskInstance: movie_pipeline.scrape backfill__2024-12-10T00:00:00+00:00 [running]> on host Jayeshs-MacBook-Air.local
[2024-12-14T09:01:53.772+0530] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='movie_pipeline' AIRFLOW_CTX_TASK_ID='scrape' AIRFLOW_CTX_EXECUTION_DATE='2024-12-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='backfill__2024-12-10T00:00:00+00:00'
[2024-12-14T09:01:53.775+0530] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-14T09:01:53.775+0530] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-14T09:01:53.776+0530] {logging_mixin.py:190} INFO - Current task name:scrape state:running start_date:2024-12-14 03:31:53.469224+00:00
[2024-12-14T09:01:53.777+0530] {logging_mixin.py:190} INFO - Dag name:movie_pipeline and current dag run status:running
[2024-12-14T09:01:53.779+0530] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-14T09:01:53.782+0530] {subprocess.py:78} INFO - Tmp dir root location: /var/folders/h6/l2cjrfls77b6_h6zs5w5vn4r0000gn/T
[2024-12-14T09:01:53.784+0530] {subprocess.py:88} INFO - Running command: ['/bin/bash', '-c', 'python controller.py --run_date 2024-12-14']
[2024-12-14T09:01:53.800+0530] {subprocess.py:99} INFO - Output:
[2024-12-14T09:01:53.897+0530] {subprocess.py:106} INFO - /usr/local/Cellar/python@3.12/3.12.2_1/Frameworks/Python.framework/Versions/3.12/Resources/Python.app/Contents/MacOS/Python: can't open file '/private/var/folders/h6/l2cjrfls77b6_h6zs5w5vn4r0000gn/T/airflowtmpmg17ulvp/controller.py': [Errno 2] No such file or directory
[2024-12-14T09:01:53.906+0530] {subprocess.py:110} INFO - Command exited with return code 2
[2024-12-14T09:01:53.922+0530] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/operators/bash.py", line 276, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2024-12-14T09:01:53.931+0530] {logging_mixin.py:190} INFO - Task instance in failure state
[2024-12-14T09:01:53.932+0530] {logging_mixin.py:190} INFO - Task start:2024-12-14 03:31:53.469224+00:00 end:2024-12-14 03:31:53.930807+00:00 duration:0.461583
[2024-12-14T09:01:53.932+0530] {logging_mixin.py:190} INFO - Task:<Task(BashOperator): scrape> dag:<DAG: movie_pipeline> dagrun:<DagRun movie_pipeline @ 2024-12-10 00:00:00+00:00: backfill__2024-12-10T00:00:00+00:00, state:running, queued_at: None. externally triggered: False>
[2024-12-14T09:01:53.933+0530] {logging_mixin.py:190} INFO - Failure caused by Bash command failed. The command returned a non-zero exit code 2.
[2024-12-14T09:01:53.935+0530] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=movie_pipeline, task_id=scrape, run_id=backfill__2024-12-10T00:00:00+00:00, execution_date=20241210T000000, start_date=20241214T033153, end_date=20241214T033153
[2024-12-14T09:01:53.961+0530] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-14T09:01:53.962+0530] {standard_task_runner.py:124} ERROR - Failed to execute job 52 for task scrape (Bash command failed. The command returned a non-zero exit code 2.; 30163)
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/operators/bash.py", line 276, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2024-12-14T09:01:54.021+0530] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-12-14T09:01:54.054+0530] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-14T09:01:54.055+0530] {local_task_job_runner.py:245} INFO - ::endgroup::
